app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="header-1">
+  <div class="back">
+    <a href="/categories" class="back-link"><span class='back-arrow'> &#8592; </span></a>
+  </div>
+  <div class="middle-title">
+    <center>
+      <h1>Transactions</h1>
+      <p>
+        Category <%= @category_name %> .
+        Total $<%= current_user.purchases.joins(:categories).where(categories: { id: params[:category_id] }).sum(:amount) %> 
+      </p>
+    </center>
+  </div>
+  <div class="next">
+    &nbsp;
+  </div>
+</div>
+
+<div class="categories">
+  <center>
+    <p style="color: green"><%= notice %></p>
+  </center>
+  <br />
+  <% @purchases.each do |purchase| %>
+    <%= render purchase %>
+  <% end %>
```

Comment:
- [ ] same here,  Kindly fix the transaction card elements placement and add the correct widths and margin values to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/68796232/228712649-2372ec27-4c80-4579-8564-432fef72933f.png) | ![image](https://user-images.githubusercontent.com/68796232/228712699-5f1fa741-f0bc-4fa9-a91e-f6450b0b9991.png) |